ferveo/benches/benchmarks/pairing.rs

```diff
@@ -39,11 +38,12 @@ pub fn lagrange(c: &mut Criterion) {
     group.bench_function("Jubjub Fr 8192*2/3 lagrange coefficients", |b| {
         b.iter(|| {
             black_box(
-                subproductdomain::SubproductDomain::<jubjub::Fr>::new(u.clone())
-                    .inverse_lagrange_coefficients()
-                    //.iter()
-                    //.map(|x| x.inverse())
-                    //.collect::<Vec<_>>(),
```

Comment:
These lines are commented out from a previous commit? If I'm understanding correctly this is an alternate benchmarking strategy that measures the inverse of each subdomain?

Reply:
These changes are from `clippy`, a Rust formatting tool. I'm going to remove commented-out code from the codebase when we figure out what it does. These lines here are probably just for the convenience of debugging/running benchmarks with different settings, so I will get rid of them now.